firmware/application/clock_manager.cpp

```diff
@@ -324,18 +333,25 @@ uint32_t ClockManager::measure_gp_clkin_frequency() {
     return get_frequency_monitor_measurement_in_hertz();
 }
 
-bool ClockManager::loss_of_signal() {
-    return hackrf_r9
-               ? clock_generator.plla_loss_of_signal()
-               : clock_generator.clkin_loss_of_signal();
+bool ClockManager::loss_of_signal(bool useRealCheck) {
+    if (hackrf_r9) {
+        if (useRealCheck) {
```

Comment:
We should probably add a comment why we are doing this: like "there are lots of out of spec portapacks that wont even boot with the correct implementation on an r9 ..."

Reply:
Good point, I will add that here